src/helpers/helpers.options.js

```diff
@@ -15,7 +15,6 @@ function toFontString(font) {
 	}
 
 	return (font.style ? font.style + ' ' : '')
-		+ (font.weight ? font.weight + ' ' : '')
```

Comment:
why this?

Reply:
This one wasn't required by TypeScript. I just noticed that `weight` was never used and thought it'd result in a smaller file to remove unused code. I can add it back if you think it's better to keep